heraldqt/ChatView/ChatBar.qml

```diff
@@ -15,8 +15,29 @@ ToolBar {
         right: parent.right
     }
 
+
     Common.Avatar {
-       displayName : Utils.try_index_or(sideBar.contactUi.currentIndex,
+
+       Connections {
```

Comment:
Is this an acceptable way of patching the RQBG gap? I would connect it to a function but this effectively does the same thing.